lib/bootloader_pvm.pm

```diff
@@ -131,6 +131,12 @@ sub enter_netboot_parameters {
     type_string_slow remote_install_bootmenu_params;
     type_string_slow " fips=1" if (get_var('FIPS_INSTALLATION'));
     type_string_slow " UPGRADE=1" if (get_var('UPGRADE'));
+
+    if (get_var('AGAMA_TEST')) {
+        my $host = get_var('OPENQA_HOSTNAME', 'openqa.opensuse.org');
+        my $iso = get_var('ISO');
```

Comment:
in general is bad practice to declare too many variables in Perl and other languages, it helps to organize a bit or debug, but in this case it is a simple replacement, you can do it in one line interpolating them, makes no difference.